### ISSUE TITLE
Missing aria-label attribute in <a> and <button> tags

### ISSUE THREAD

**Issue by minn09** [minn09/portfolio#4]
![image](https://github.com/user-attachments/assets/f815b7e4-3ff0-488f-acd7-dc418739d370)
![image](https://github.com/user-attachments/assets/bdb9361d-d3fe-464a-a90e-c602ca465b77)

**Comment by minn09** [comment]
![image](https://github.com/user-attachments/assets/90ae1f7d-9fb6-4ab1-b394-02e8e6211c6c)
![image](https://github.com/user-attachments/assets/da8f482d-8f8d-487c-9b0f-4c4311be527b)
